docs/en_US/Compressor/Framework.md

```diff
@@ -1,15 +1,33 @@
 # Design Doc
 
 ## Overview
-The model compression framework has two main components: `pruner` and `module wrapper`.
+Following example shows how to use a pruner:
+```python
+from nni.compression.torch import LevelPruner
 
-### pruner
-A `pruner` is responsible for :
-1. provide a `cal_mask` method that calculates masks for weight and bias.
-2. replace the module with `module wrapper` based on config.
-3. modify the optimizer so that the `cal_mask` method is called every time the `step` method is called.
+# load a pretrained model or train a model before using a pruner
+
+configure_list = [{
+    'sparsity': 0.7,
+    'op_types': ['Conv2d', 'Linear'],
+}]
+
+optimizer = torch.optim.SGD(model.parameters(), lr=0.001, momentum=0.9, weight_decay=1e-4)
+pruner = LevelPruner(model, configure_list, optimizer)
+model = pruner.compress()
+
+# model is ready for pruning, now start finetune the model,
+# the model will be pruned during training automatically
+```
+
+A pruner receives model, config_list and optimizer as arguments. It prunes the model per user defined configuratioin (defined by `config_list`) during training loop by adding a hook on `optimizer.step()`.
```

Comment:
configuratioin -> configuration.

Reply:
I don't understand this sentence. What does `model per user defined configuratioin` mean?

Reply:
Thanks, sentence updated.